src/app/georeferencer/qgsgeorefmainwindow.cpp

```diff
@@ -1176,6 +1180,7 @@ void QgsGeoreferencerMainWindow::setupConnections()
 
   // Connect mapCanvas rotation widget
   connect( mRotationEdit, static_cast < void ( QgsDoubleSpinBox::* )( double ) > ( &QgsDoubleSpinBox::valueChanged ), this, &QgsGeoreferencerMainWindow::updateCanvasRotation );
+  connect( mCanvas, &QgsMapCanvas::destinationCrsChanged, this, &QgsGeoreferencerMainWindow::invalidateCanvasCoords );
```

Comment:
```suggestion
  connect( QgisApp::instance()->mapCanvas(), &QgsMapCanvas::destinationCrsChanged, this, &QgsGeoreferencerMainWindow::invalidateCanvasCoords );
```

Reply:
Done thanks! Though it's really weird that only for this one using the cached canvas pointer doesn't work...

Reply:
`mCanvas` is not the main map canvas, it's the georef window canvas, that's why!